GameData/SPEngine/Parts/A4_proc.cfg

```diff
@@ -12,7 +12,21 @@
 	!engineType = // prevent ROE from deleting us
 }
 
-+PART[Bumper_Engine_Unclad]:AFTER[ZREALPLUME]
++PART[Bumper_Engine_Unclad]:NEEDS[!ROEngines]:AFTER[ZREALPLUME]
+{
+	@name = SPEngineA4Unclad
+	@title = A-class Procedural Engine (Unclad)
+	@description = Alcohol/LOx atmospheric engine, like Aggregat or RD-100 series.  Without enclosing boattail.
+	MODULE
+	{
+		name = ModuleSPEngine
+		familyLetter = A
+	}
+	@identicalParts = Bumper_Engine,Bumper_Engine_Unclad
```

Comment:
Hmm, not your fault 'cos you just copied what was there already, but these `@identicalParts`es are wrong.  They should be `SPEngineA4,SPEngineA4Unclad`, on all three `+PART[]`s.